bindings/nodejs/examples/how_tos/outputs/features.ts

```diff
@@ -53,15 +53,17 @@ async function run() {
         const nftOutputWithMetadata = await client.buildNftOutput({
             nftId: '0x0000000000000000000000000000000000000000000000000000000000000000',
             unlockConditions: [addressUnlockCondition],
-            features: [new MetadataFeature(utf8ToHex('Hello, World!'))],
+            features: [
+                new MetadataFeature({ data: utf8ToHex('Hello, World!') }),
+            ],
         });
 
         // Output with immutable metadata feature
         const nftOutputWithImmutableMetadata = await client.buildNftOutput({
             nftId: '0x0000000000000000000000000000000000000000000000000000000000000000',
             unlockConditions: [addressUnlockCondition],
             immutableFeatures: [
-                new MetadataFeature(utf8ToHex('Hello, World!')),
+                new MetadataFeature({ data: utf8ToHex('Hello, World!') }),
```

Comment:
Is this correct? No quotes on `data`?

Reply:
Yes, I even added the quotes initially but they get removed from format

Reply:
That's so weird, it feels wrong. Did you try it by any chance?

Reply:
That's normal in JS world `console.log(JSON.stringify({data: "0x..."}))` ->`'{"data":"0x..."}'`

Reply:
Also tried the example `Running the feature examples in nodejs and python` with the feature

Reply:
Alright, I guess I'll never understand JS